public_html/php/oauth.php

```diff
@@ -94,7 +94,7 @@ function setupSession() {
 		$params = session_get_cookie_params();
 		$lifetime = $params['lifetime'];
 		if ( $this->cookie_lifetime !== null ) {
-			$lifetime = $this->cookie_lifetime();
+			$lifetime = call_user_func($this->cookie_lifetime);
```

Comment:
Apparently you cannot call a property directly so we need to do it like this.

Reply:
I'm trying to follow here and failing to see where the lifetime of the cookie is set by the caller but why do we expect this to be a function and not a simple value?